organisaatio-ui/src/components/Sivut/LomakeSivu/LomakeSivu.tsx

```diff
@@ -149,26 +170,56 @@ const LomakeSivu = (props: any) => {
         );
     }
 
+    const accordionProps = () => {
+        const lomakkeet = [] as React.ReactElement[];
+        const otsikot = [] as string[];
+        lomakkeet.push(
+            <PerustietoLomake
+                setYtjDataFetched={setYtjDataFetched}
+                handleOnChange={handleOnChange}
+                organisaatioTyypit={organisaatioTyypit}
+                organisaatio={organisaatio}
+                language={language}
+                maatJaValtiot={maatJaValtiot}
+                opetuskielet={oppilaitoksenOpetuskielet}
+            />
+        );
+        otsikot.push(i18n.translate('LOMAKE_PERUSTIEDOT'));
+        if (organisaatio.yhteystiedot) {
+            lomakkeet.push(
+                <YhteystietoLomake handleOnChange={handleOnChange} yhteystiedot={organisaatio.yhteystiedot} />
+            );
+            otsikot.push(i18n.translate('LOMAKE_YHTEYSTIEDOT'));
+        }
+        lomakkeet.push(<NimiHistoriaLomake nimet={organisaatio.nimet} />);
+        otsikot.push(i18n.translate('LOMAKE_NIMIHISTORIA'));
+
+        if (organisaatio.oid !== ROOT_OID) {
+            lomakkeet.push(<OrganisaatioHistoriaLomake oid={organisaatio.oid} />);
+            otsikot.push(i18n.translate('LOMAKE_RAKENNE'));
+        }
+
+        return { lomakkeet: lomakkeet, otsikot: otsikot };
+    };
+
     return (
         <PohjaSivu>
             <div className={styles.YlaBanneri}>
                 <div>
-                    <a href="/">
+                    <Link to="/">
                         <Icon icon={homeIcon} />
-                    </a>
+                    </Link>
                 </div>
                 {organisaatioNimiPolku.map((o, index) => [
                     <div>
-                        <a href={`/organisaatio/lomake/${o.oid}`}>
-                            {o.nimi[language] || o.nimi['fi'] || o.nimi['sv'] || o.nimi['en']}
-                        </a>
+                        <Link to={`${o.oid}`}>{o.nimi[language] || o.nimi['fi'] || o.nimi['sv'] || o.nimi['en']}</Link>
                     </div>,
                     organisaatioNimiPolku.length - 1 !== index && <div> &gt; </div>,
                 ])}
             </div>
             <div className={styles.ValiContainer}>
                 <div className={styles.ValiOtsikko}>
-                    <h3>{organisaatio.tyypit[0]}</h3>
+                    <h3>{organisaatio.tyypit ? organisaatio.tyypit[0] : 'N/A'}</h3>
```

Comment:
hardcoded 'N/A'?

Reply:
true, should "never" end up there though